### PR TITLE
Update truncate logic

### PR DESCRIPTION
**Pull request by toni-sharpe**
Truncate needs CASCADE to avoid "cannot truncate a table referenced in a foreign key constraint" errors

Following this: https://stackoverflow.com/a/56700574

Fixes https://github.com/nearform/titus/issues/1534

### CODE REVIEW

File: packages/titus-db-manager/truncate/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = async function truncate(pg, { schema, logger }) {
   await pg.query(`
     TRUNCATE TABLE
       ${schema}.some_table
-    RESTART IDENTITY;
+    RESTART IDENTITY CAASCADE;
```
Comment (simoneb) [inline packages/titus-db-manager/truncate/index.js:7]:
```suggestion
    RESTART IDENTITY CASCADE;
```

**Comment by simoneb** [comment]
@toni-sharpe do you have any capacity to fix the failing tests? it should be fairly straightforward

**Comment by simoneb** [comment]
@toni-sharpe ping

**Comment by toni-sharpe** [comment]
@simoneb I did the tests and the build is green here now, thanks for the ping!

(I'll be away 'til 01.08 after today)

**Review by simoneb — APPROVED** [review]

**Pull request merged**